cli/login.go

```diff
@@ -267,12 +275,66 @@ func (r *RootCmd) login() *serpent.Command {
 					trial = v == "yes" || v == "y"
 				}
 
+				if trial {
+					if firstName == "" {
```

Comment:
suggestion: we can instead just leverage the zero value and assign directly to the fields of TrialInfo

e.g.

```
var trialInfo codersdk.CreateFirstUserTrialInfo
if trial {
  if trialInfo.FirstName == "" {
    ...
  }
  ...
}
```